apps/backend/src/lambdas/helpers.ts

```diff
@@ -0,0 +1,60 @@
+import { parse, subBusinessDays } from 'date-fns';
+import { Ministries } from '../constants';
+
+export const setReconciliationDates = (
+  event: unknown,
+  maxNumDaysToReconcile: number
```

Comment:
This seems like it's more `numDaysToReconcile` than a `max` of some sort, am I misunderstanding?